plex/config.json

```diff
@@ -8,6 +8,10 @@
   "startup": "services",
   "arch": ["aarch64", "amd64", "armv7", "i386"],
   "map": ["media:rw", "share:rw", "ssl"],
+  "devices": [
+    "/dev/dri/renderD128:/dev/dri/renderD128",
+    "/dev/dri/card0:/dev/dri/card0"
+  ],
```

Comment:
This will go south on devices that don't have this endpoints.

Reply:
Sorry, you can delete this if it is possible. It should have gone to my private repo.